### PR TITLE
Clarify what happens when no bootnodes are specified. 

### PR DESCRIPTION
**Pull request by Gabriel-Trintinalia**

### CODE REVIEW

File: versioned_docs/version-23.10.3/private-networks/how-to/configure/bootnodes.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ By default, peer discovery listens on all available network interfaces. If the d
 
 ## Configure bootnodes in a production network
 
-A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network.
+A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network. In case of no bootnodes, Mainnet default bootnodes will be used.
```
Comment (macfarla) [inline versioned_docs/version-23.10.3/private-networks/how-to/configure/bootnodes.md:45]:
```suggestion
A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network. If you don't configure any bootnodes, Mainnet default bootnodes will be used.
```

**Review by macfarla — APPROVED** [review]
suggestion on wording

File: versioned_docs/version-23.10.3/private-networks/how-to/configure/bootnodes.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ By default, peer discovery listens on all available network interfaces. If the d
 
 ## Configure bootnodes in a production network
 
-A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network.
+A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network. If you don't configure any bootnodes, Mainnet default bootnodes will be used.
```
Comment (alexandratran) [inline versioned_docs/version-23.10.3/private-networks/how-to/configure/bootnodes.md:45]:
```suggestion
A network must have at least one operating bootnode. To allow for continuity in the event of failure, configure two or more bootnodes in a production network. If you don't configure any bootnodes, Besu uses Mainnet's default bootnodes.
```

**Review by alexandratran — COMMENTED** [review]
Wording suggestion. Also, can you apply this update to the latest version of the docs located in the `/docs/private-networks` folder?

File: versioned_docs/version-23.10.3/private-networks/how-to/configure/bootnodes.md (hunk 2 above)
Comment (Gabriel-Trintinalia) [reply versioned_docs/version-23.10.3/private-networks/how-to/configure/bootnodes.md]: Done! Thanks @alexandratran

**Review by alexandratran — APPROVED** [review]

**Pull request merged**